<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewImageDialog</class>
 <widget class="QDialog" name="NewImageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>245</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Image...</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxSize">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>44</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="currentText">
    <string>128</string>
   </property>
   <property name="currentIndex">
    <number>5</number>
   </property>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>32</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>64</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>128</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>256</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelSize">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>67</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Size:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelName">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>86</y>
     <width>67</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Name: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditFileName">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelI3C">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>86</y>
     <width>67</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>.i3c</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPixels_Side">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Pixels/side</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPath">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>136</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Path:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPath">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>140</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPath">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewImageDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewImageDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
